wpeverest/everest-forms
# Refactor - Email attachment workflow.

### All Submissions:






### Changes proposed in this Pull Request:

The field value is currently not utilized in email attachment post processing latchers. Instead, the entry variable is used, which causes problems specifically in entry being disabled. The formatting goes awry, and there are several inconsistencies in the data that is passed and parsed by expectant filters.

This PR intends to slowly shift away from html_field_values filter, which would require additional queries, or inefficient data queries just to be able to format data consistently. Rather than the loop making edge cases and per-formatting data, delegate it to an exporter function which already expects the data..

Closes # .

### How to test the changes in this Pull Request:

1. Make a form with the form builder.
2. Upon form submission, notice no formatting issues/inconsistencies whatsoever.
3. Upon disabling entry, notice the outputted data is exactly as it were if entry wasn't disabled.

### Types of changes:




### Other information:




### Changelog entry

> Refactor - Email attachment workflow.
